testing/regress/ecl/aggidx1.ecl

```diff
@@ -24,6 +24,8 @@ useSequential := #IFDEFINED(root.useSequential, false);
 
 //--- end of version configuration ---
 
+#option ('obfuscateOutput', true);
```

Comment:
Sorry, but we have to be pragmatic. Making the regression suite bigger just because it feels right means making the regression suite less likely to be run. In the absence of coverage metrics we have to rely on common sense to know when a test is adding something and when it is not. We have 12000 closed Jiras - there are 500ish tests in the regression suite...